ARCs/arc-0034.md

```diff
@@ -0,0 +1,124 @@
+title: Governance proposal process
+description: Criteria for the creation of proposals.
+author: Stéphane Barroso (@SudoWeezy), Adriana Belotti, Massimo Morini, Michel Treccani, John Woods, Shai Halevi
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/
+status: Draft
+type: Meta
+created: 2022-11-22
+---
+ 
+## Abstract
+
+The goal of this proposal is to increase transparency and efficiency in the decision-making process by clearly defining the steps involved and ensuring that all proposals are given proper consideration. 
+ 
+## Specification
+The key words "**MUST**", "**MUST NOT**", "**REQUIRED**", "**SHALL**", "**SHALL NOT**", "**SHOULD**", "**SHOULD NOT**", "**RECOMMENDED**", "**MAY**", and "**OPTIONAL**" in this document are to be interpreted as described in <a href="https://www.ietf.org/rfc/rfc2119.txt">RFC-2119</a>.
+ 
+
+### What is a proposal
+A proposal within the xGov program aims to provide funding for:
+- Funding for early stage projects
+- Facilitation of community involvement
+- Addressing the needs of the community
+ 
+### Duty of a proposer
+Having the ability to propose measures for a vote is a significant power, which requires:
+- A thorough understanding of the needs of the community
+- Alignment of personal interests with the advancement of the Algorand ecosystem.
+> Issues requiring resolution may have been discussed on various online platforms such as forums, discord, and social media networks.
+
+
+### Voting System
+The voting system for the xGov program will be the following:
+- Only members of xGov are eligible to participate in the voting process
+- Utilization of the Algo commited by xGov as a form of voting currency, with a 1:1 ratio of Algos to votes
+- Proposals that do not meet the minimum threshold of votes required will be removed from consideration.
+
+
+### Life of a proposal
+The proposal process will have the same duration as the governance period. Within the 3 months, the following steps will be implemented:
+- Anyone can submit a proposal at any time. Proposals will be hidden to prevent bias.
+- 0.5 months after the start of a governance periode, proposals submitted will be revealed. 
+- 1 month is allocated for proposals refinement and temperature check. Proposals will be evaluated and refined by the community and xGovs.
+- 1 month is allocated for voting on proposals. The community will vote on proposals that have passed the refinement and temperature check stage.
+- 0.5 months for negotiation and claiming grants. Proposals that have been approved will be negotiated and grants will be claimed.
+
+### Submit a proposal
+In order to submit a proposals, a proposer needs to fill a form on the website provided for this purpose: <a href="">URL TBD</a>.
+
+#### Information requried to submit a proposal
+``` json 
+{
+   "Title": "title of the proposal",
+   "ProposerEmail": "",
+   "Country": "",
+   "CompanyName":"",
+   "Team": [],
+   "ExperienceOnAlgorand": {"Low", "Medium", "High" ),
```

Comment:
```suggestion
   "ExperienceOnAlgorand": ["Low", "Medium", "High"],
```
@SudoWeezy i think you might have some json typos now that i took a look at it.  Arrays should use `[ ]`